### ISSUE TITLE
question about model evaluation using scripts or commands

### ISSUE THREAD

**Issue by jinz2014** [YerevaNN/mimic3-benchmarks#76]
Hi Developer,

I am confused by the Evaluation described below and testing the model after it is trained.  I am not sure which ways we can test the model after it is trained.

In your tutorial, the following command is used to train the model (e.g. ihm):
python -um mimic3models.in_hospital_mortality.main --network mimic3models/keras_models/lstm.py --dim 16 --timestep 1.0 --depth 2 --dropout 0.3 --mode train --batch_size 8 --output_dir mimic3models/in_hospital_mortality

Can we test the model in the following way ?
python -um mimic3models.in_hospital_mortality.main --network mimic3models/keras_models/lstm.py --dim 16 --timestep 1.0 --depth 2 --dropout 0.3 --mode test --batch_size 8 

---
Evaluation
For each of the four tasks we provide scripts for evaluating models. These scripts receive a csv file containing the predictions and produce a json file containing the scores and confidence intervals for different metrics. We highly encourage to use these scripts to prevent any mistake in the evaluation step. For details about the usage of the evaluation scripts view the mimic3benchmark/evaluation/README.md file.

**Comment by EmmaRocheteau** [comment]
That looks similar to what I've been doing. Presumably after running that and retrieving your test predictions you'd then run the script mimic3benchmark.evaluation.evaluate_ihm.py script

**Comment by Harhro94** [comment]
@jinz2014, As @EmmaRocheteau correctly noted you need to get the prediction file using the `--test` mode and then feed that file to the corresponding evaluation script.

**Comment by syfantid** [comment]
Another important point. When using the --test mode, don't forget to add the --load_state argument to specify the Keras model state you want to load, e.g.:

`python -um mimic3models.in_hospital_mortality.main --network mimic3models/keras_models/lstm.py --dim 16 --timestep 1.0 --depth 2 --dropout 0.3 --mode test --batch_size 8 --load_state mimic3models/in_hospital_mortality/keras_states/k_lstm.n16.d0.3.dep2.bs8.ts1.0.epoch28.test0.28153726805873275.state --output_dir mimic3models/in_hospital_mortality`
